Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +35,241 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            fontSize='1.7vh'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            fontSize='1.7vh'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
                     </Flex>
                 </Flex>
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 1 Temp'
-                    data={props.data?.string1_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    py='0.5rem'
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    borderBottom='0.2vh'
+                    //border='100vh'
+                    //gap={0.5}
+                >
+
+                    <GridItem
+                        border="5px solid transparent"
+                        colStart={2}
+                        bg='headerBg'
```

Comment:
```suggestion
                        bg={headerBg}
```
Sorry, didn't specify. `headerBg` is a variable that has to be set using `getColorMode()` and `getColor()` (see [`BatteryPack.js`](https://github.com/badgerloop-software/chase-car-dashboard/blob/main/Frontend/src/Components/BatteryCells/BatteryPack.js))

Reply:
I am not sure what you think but what if I do not set the background color at all because the component seems to look fine like that as well